src/config/nodemailerConfig.ts

```diff
@@ -0,0 +1,33 @@
+import * as dotenv from "dotenv";
+import nodemailer from "nodemailer";
+
+dotenv.config();
+
+const NODEMAILER_EMAIL = String(process.env.NODEMAILER_EMAIL);
+const NODEMAILER_PASS = String(process.env.NODEMAILER_PASS);
+
+const transporter = nodemailer.createTransport({
+    service: "gmail",
+    auth: {
+        user: NODEMAILER_EMAIL,
+        pass: NODEMAILER_PASS,
+    },
+});
+
+export const sendConfirmEMail = async (
```

Comment:
Can you revamp this to use sendgrid instead of nodemailer?

Reply:
I will share the credentials later today

Reply:
Alright noted. I'll be expecting the credentials